frontend/src/components/Avatar/Avatar.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  Avatar as ChakraAvatar,
+  AvatarBadge as ChakraAvatarBadge,
+  AvatarProps as ChakraAvatarProps,
+  Box,
+  ButtonProps,
+  Icon,
+  MenuItemProps,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+
+import { BxsUser } from '~/assets/icons/BxsUser'
+
+import DropdownMenu from '../Menu'
+
+export interface AvatarProps extends ChakraAvatarProps {
+  hasNotification?: boolean
+}
+
+/*
+ * Utility to extract first letter of avatar name
+ * Override default getInitials Avatar method
+ */
+const getInitials = (fullName: string) => fullName[0]
+
+/*
+ * Avatar component
+ * May be wrapped in AvatarMenuButton
+ */
+export const Avatar = (props: AvatarProps): JSX.Element => {
+  const avatarBadge = props.hasNotification ? <ChakraAvatarBadge /> : ''
+
+  return (
+    <ChakraAvatar getInitials={getInitials} {...props}>
```

Comment:
Would there not be a warning from React that some unknown `hasNotification` prop is being passed into the dom? We should destructure if we do not want to pass it into ChakraAvatar:

```tsx
const Avatar = ({hasNotification, ...rest}) => {
...
}
```

Reply:
thanks, excluded hasNotification